components/AsanaCard/index.js

```diff
@@ -0,0 +1,36 @@
+//import Image from "next/image";
+import styled from "styled-components";
+
+export default function AsanaCard({asana}) {
+  console.log(asana.image);
```

Comment:
Please delete this console log, inside your final PR should be no console logs.

Reply:
Oh thank, i forgot it and change it. should i do a new PR?